src/table/table.scss

```diff
@@ -4,383 +4,404 @@
+        @include themed {
+          border-top-color: t(iui-color-foreground-primary);
         }
       }
-    }
 
-    &.iui-tables-row-active {
-      @include themed {
-        border: solid 1px t(iui-color-foreground-primary);
-        background: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-5));
+      + .iui-expandable-content {
+        @include themed {
+          border-color: transparent t(iui-color-foreground-primary);
+        }
       }
     }
+    // #endregion Selection
 
-    &.iui-tables-row-new {
+    // #region Statuses & states
+    &.iui-new {
       font-weight: $iui-font-weight-semibold;
-
-      > .iui-tables-filename {
-        position: relative;
-      }
-
-      > .iui-tables-filename::before {
-        @include themed {
-          color: t(iui-color-foreground-positive);
-        }
+      
+      > .iui-main-column::before {
         content: '•';
         position: absolute;
         left: 0;
         font-size: $iui-font-size-title;
+        @include themed {
+          color: t(iui-color-foreground-positive);
+        }
       }
     }
 
-    &.iui-tables-row-disabled {
+    &.iui-disabled,
+    &.iui-disabled + .iui-expandable-content {
+      font-style: italic;
+      cursor: not-allowed;
       @include themed {
         color: t(iui-text-color-muted);
       }
-      font-style: italic;
-      cursor: not-allowed;
-
-      &:hover {
-        @include themed {
-          background-color: t(iui-color-background-1);
-        }
 
-        > .iui-tables-slot > .iui-tables-menu {
-          visibility: hidden;
-        }
+      &:hover > .iui-slot > .iui-more-options {
+        visibility: hidden;
       }
 
-      > .iui-tables-cell {
-        > img,
-        > svg,
-        > .iui-user-icon,
-        > .iui-user-icon > img {
-          filter: $iui-icons-color-multicolor-disabled;
-        }
+      img,
+      svg:not(.iui-radial),
+      .iui-user-icon {
+        filter: $iui-icons-color-multicolor-disabled;
       }
     }
+
+    &.iui-positive {
+      @include iui-table-row-status($status:positive);
+    }
+    
+    &.iui-warning {
+      @include iui-table-row-status($status:warning);
+    }
+
+    &.iui-negative {
+      @include iui-table-row-status($status:negative);
+    }
+    // #endregion Statuses & states
   }
 
-  .iui-tables-message-container {
+  // Empty & loading states
+  &.iui-message {
+    text-align: center;
+    height: 100%;
+    padding: $iui-xl;
     @include themed {
       color: t(iui-text-color-muted);
       background-color: t(iui-color-background-2);
     }
-    display: flex;
-    justify-content: center;
-    height: 100%;
-    padding: $iui-xl;
   }
+}
 
-  .iui-tables-paginator {
-    @include themed {
-      background-color: t(iui-color-background-3);
+@mixin iui-table-cell {
+  align-items: center;
+  display: flex;
+  flex-grow: 1;
+  min-width: $iui-l * 6;
+  min-height: $iui-baseline * 5;
+  padding: 0 $iui-m;
```

Comment:
Can we change to only `padding-left: $iui-m;` and no padding-right ?
Content get truncated when there should be enough place to display it because of padding-right. 
Because there is padding-left on the next column, the content doesn't look bad at all.
![image](https://user-images.githubusercontent.com/4070713/121257569-d8470e00-c87b-11eb-9412-72725b9055dd.png)

Reply:
I think padding is fine.

This would look terrible without padding-right:
![image](https://user-images.githubusercontent.com/9084735/121257886-3a077800-c87c-11eb-96d2-bd6cfdbab978.png)

Reply:
Also, reducing padding-right would be better than removing it.

Reply:
Maybe reducing padding by half on both side would be an alternative. Though the reason I'm saying to none is because you can clearly see that the text could fit in this cell but the table doesn't want you to do it anyway and that's why I suggest to remove it all. So the end of the cell is where the text get truncated and not before.

Reply:
Looks like we could go as low as 12px padding (at least on the left side).  Any smaller and we run into this issue where the new dot gets too close to the filetype icons:
![Screen Shot 2021-06-09 at 12 14 21 PM](https://user-images.githubusercontent.com/849817/121391315-3a575000-c91c-11eb-9233-52eadd338a08.png)

Reply:
Changed